packages/inertia-react/index.d.ts

```diff
@@ -86,3 +86,29 @@ export interface InertiaFormProps<TForm = Record<string, any>> {
 }
 
 export function useForm<TForm = Record<string, any>>(initialValues: TForm): InertiaFormProps<TForm>;
+
+export interface AppSetupPageProps {
+    initialPage: Inertia.PageProps,
+    initialComponent: any,
+    resolveComponent: (name: string) => Promise<any>,
+    titleCallback: (title: string) => string,
+    onHeadUpdate: ((elements: any[]) => boolean) | null,
```

Comment:
```suggestion
    onHeadUpdate: (elements: string[]) => void,
```
See the `onUpdate` parameter at https://github.com/inertiajs/inertia/blob/master/packages/inertia/src/head.ts#L56